README.md

```diff
@@ -30,6 +30,10 @@ cd <local folder>
 - Twitter: [@asia_magana](https://twitter.com/asia_magana)
 - LinkedIn: [Asia Magana](https://www.linkedin.com/in/asia-magana-60b451200/)
 
+👤 **Assad Isah**
+
+- GitHub: [@nottherealalanturing](https://github.com/nottherealalanturing)
```

Comment:
- [ ] In the README Please write the correct steps to clone and tun the project. Currently, the repo which you have given in the git clone is wrong. Also, add the steps to run the file as well. [ Hint: `ruby list.rb`].